openSUSEway.spec

```diff
@@ -61,10 +60,8 @@ Suggests:       vifm
 Suggests:       vim
 Suggests:       imv
 
-# basic multi-media
-Requires:       clipman
```

Comment:
As per discussion in #184, I would like to change this to `Recommends` instead of dropping the `Requires`